content/docs/main-concepts/api-clients.mdx

```diff
@@ -0,0 +1,67 @@
+
+These locks are OAuth 2.0 Grant Types and they keys are the credentials needed to gain access to the data behind each API Client. The API Client used to gain access and other information about the user is encoded into an Access Token to be used on each subsequent request. This is the "wristband" that let's developers and the OrderCloud team know where and how a requestee gained access to the data.
+
+## Creating API Clients
+Like anything else on the OrderCloud platform, API Clients can be managed via our RESTful API. You may ask, "how can I create an API Client if one is required to access the API?". Whenever a new organization is created on OrderCloud, a "default" API Client is created along with a Full Access Seller user. You can use this initial context to create anything else in OrderCloud, including more API Clients.
+
+### Naming Your API Client
+Choosing the right name for your API Client is important. By picking something descriptive, future developers will better understand when they should be utilized.
+
+### Defining User Access
+You can control which parties in your organization can access each of your API Clients using any combination of the following three properties:
+
+- `AllowSeller` - All Seller Users can use the API Client
+- `AllowAnySupplier` - Supplier Users in _any_ Supplier company can use the API Client
+- `AllowAnyBuyer` - Buyer Users in _any_ Buyer company can use the API Client
+
+Creating an API Client assignment enables further control, for instance, allowing a specific Buyer company access while blocking others. Each assignment references an API Client ID paired with a Buyer or Supplier ID. `AllowAnyBuyer` and `AllowAnySeller` will nullify direct assignments for their respective company type, so be sure set those properties to `false` if you need that level of control.
+
+### Client Secrets and OAuth
+Adding a Client Secret to an API Client changes how you'll interact with the OrderCloud OAuth server. If a Client Secret exists, it is required on all OAuth grant types (yes, even Password). 
+
+Normally this configuration is used for automated systems that require access to your OrderCloud data. This is because it enables the use of OAuth's **Client Credentials** grant type. 
+
+You will quickly realize that a Client Secret by itself is not sufficient enough for OrderCloud to grant you access to the API. Every user in OrderCloud can have their own unique level of data access. In order to know _what_ an authenticated party can read from or write to, every interaction must come from a **User Context**.
+
+The `DefaultContextUserName` property is for exactly that. It let's OrderCloud know what user context to provide when authentication occurs using the Client Credentials grant type. As far as OrderCloud knows, you *are* the default context user when using this grant type.
+
+Read more about supported OAuth workflows here.
```

Comment:
I'm assuming this link doesn't exist yet, which is why it's not here - any ideas on how to make sure that we flag all of these so that they get updated prior to go live?

Reply:
Actually I think that does exist, just not linked. I've been adding TODO: comments which makes it fairly easy to scan for. DJ did a round of checking last week and I'll probably do another one soon as well